src/background.js

```diff
@@ -3,7 +3,7 @@ import 'src/omnibar'
 
 
 function openOverview() {
-    browser.tabs.create({
+    browser.tabs.update({
```

Comment:
I agree with your preference of opening it in a new tab by default. Much more intuitive and as you said, does not disturb the current work flows.